### PR TITLE
Adding pagination component

### PR DESCRIPTION
**Pull request by IanWoodard**
Added a nearly fully functional pagination component. The only thing left to do is add corresponding logic for retrieving pool information that supports pagination or simply truncating the results. Additionally, as of now, the pagination is not accurate with respect to the contents of the page simply for testing purposes and can easily be changed to reflect the contents of the page.

As usual, I added some images of the components below:

page 1, 10 results per page, 100 results
pagination1

page 4, 10 results per page, 100 results
pagination3

page 10, 10 results per page, 100 results
pagination4

page 1, 20 results per page, 100 results
pagination2

### CODE REVIEW

**Comment by IanWoodard** [comment]
Another thing to note is that the dropdown is not to spec. This is done on purpose as the dropdown component currently has a PR of its own and will be added to this once it is merged.

**Comment by haydenshively** [comment]
LGTM!

File: src/components/common/AppBody.ts [hunk 1]
```diff
@@ -4,4 +4,5 @@ import tw from 'twin.macro';
 export default styled.div`
   ${tw`bg-grey-50 w-full h-full text-white flex flex-col min-h-screen py-16`}
   font-family: 'Inter';
+  background-color: rgba(7, 14, 18, 1);
```
Comment (adamegyed) [inline src/components/common/AppBody.ts]: (Probably) drop from this PR and include only in the header/styling PR.

File: src/components/common/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  line-height: 24px;
+  width: 40px;
+  height: 40px;
+
+  &.active {
+    color: rgba(82, 182, 154, 1);
+  }
+`;
+
+const Ellipsis = styled.span`
```
Comment (adamegyed) [inline src/components/common/Pagination.tsx]: Rename to `EllipsisWrapper` or similar to indicate that this is not itself the ellipsis, but a styling span for it.

**Review by adamegyed — APPROVED** [review]
LGTM! Just had 2 nits.

**Pull request merged**